flexget/ui/plugins/movies/movies.py

```diff
@@ -44,8 +44,8 @@ def index():
             # this is probably not needed since non cached movies are retried also
             # in the cover function
             #
-            #import thread
-            #thread.start_new_thread(tmdb_lookup, (), {'imdb_id': item.imdb_id})
+#            #import thread
```

Comment:
PEP8 yells about the "#" Not being at the beginning of the line only when it's actual Python code after the "#". Do not ask me I dunno so I fixed it to where it didn't show up.